app/controllers/projects/samples/transfers_controller.rb

```diff
@@ -3,13 +3,14 @@
 module Projects
   module Samples
     # Controller actions for Project Samples Transfer
-    class TransfersController < Projects::ApplicationController
+    class TransfersController < Projects::SamplesController
```

Comment:
**nitpick (blocking):** Revert this change and move `load_samples` into `Projects::ApplicationController`.

I think you did this to get access to `load_samples`, but that could be added to the application controller instead. As other wise a lot more logic is being added then is necessary.

Reply:
I think this is addressed in e1ef33a